### ISSUE TITLE
When trying to create a file : Uncaught TypeError: Cannot read property 'remote' of undefined

### ISSUE THREAD

**Issue by pooreal** [icetee/remote-ftp#522]
[Enter steps to reproduce below:]
1. Right click on the folder you want to create a file within
2. Click Add file , type a name and press enter.

**Atom Version**: 1.11.2
**Electron Version**: 0.37.8
**System**: Microsoft Windows 7 Ultimate 
**Thrown From**: [Remote-FTP](https://github.com/mgrenier/remote-ftp) package, v0.9.1
### Stack Trace

Uncaught TypeError: Cannot read property 'remote' of undefined

```
At /C:/Users/Pooria/.atom/packages/Remote-FTP/lib/menus/commands.js:197

TypeError: Cannot read property 'remote' of undefined
    at space-pen-div.<anonymous> (C:/Users/Pooria/.atom/packages/Remote-FTP/lib/menus/commands.js:175:38)
    at space-pen-div.jQuery.event.dispatch (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:4435:9)
    at space-pen-div.elemData.handle (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:4121:28)
    at Object.jQuery.event.trigger (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:4350:12)
    at space-pen-div.<anonymous> (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:4901:17)
    at Function.jQuery.extend.each (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:374:23)
    at AddDialog.jQuery.fn.jQuery.each (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:139:17)
    at AddDialog.jQuery.fn.extend.trigger [as originalTrigger] (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\jquery\dist\jquery.js:4900:15)
    at AddDialog.$.fn.trigger (C:\Users\Pooria\.atom\packages\Remote-FTP\node_modules\space-pen\lib\space-pen.js:606:19)
    at AddDialog.module.exports.AddDialog.onConfirm (C:\Users\Pooria\.atom\packages\Remote-FTP\lib\dialogs\add-dialog.js:22:8)
    at space-pen-div.atom.commands.add.core:confirm (C:\Users\Pooria\.atom\packages\Remote-FTP\lib\dialogs\dialog.js:51:10)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\Pooria\AppData\Local\atom\app-1.11.2\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\Pooria\AppData\Local\atom\app-1.11.2\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\Pooria\AppData\Local\atom\app-1.11.2\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\Pooria\AppData\Local\atom\app-1.11.2\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (C:\Users\Pooria\AppData\Local\atom\app-1.11.2\resources\app.asar\src\window-event-handler.js:114:36)
    at HTMLDocument.<anonymous> (C:\Users\Pooria\AppData\Local\atom\app-1.11.2\resources\app.asar\src\window-event-handler.js:3:61)
```
### Commands

```
     -7:39 core:undo (atom-text-editor.editor.is-focused)
  5x -7:36.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -7:16.3.0 autoclose-html:close-and-complete (atom-text-editor.editor.is-focused)
     -7:15.6.0 core:save (atom-text-editor.editor.is-focused)
     -7:07.9.0 remote-ftp:add-file (div.header.list-item)
     -7:02.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -6:52 remote-ftp:connect (atom-text-editor.editor.is-focused)
     -6:33 remote-ftp:add-file (div.header.list-item)
     -6:25.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
 32x -6:00.9.0 core:undo (atom-text-editor.editor.is-focused)
     -5:55.8.0 core:save (atom-text-editor.editor.is-focused)
     -5:53.4.0 remote-ftp:add-file (span.name.icon.icon-file-directory)
     -5:50.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:12.1.0 core:save (atom-text-editor.editor.is-focused)
     -0:08.0 remote-ftp:add-file (span.name.icon.icon-file-directory)
     -0:06.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "telemetryConsent": "limited",
    "themes": [
      "seti-ui",
      "one-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
an-old-hope-syntax, v0.5.0 (inactive)
autoclose-html, v0.23.0 (active)
color-picker, v2.2.2 (active)
minimap, v4.25.6 (active)
minimap-find-and-replace, v4.5.1 (active)
monokai-seti, v0.7.0 (inactive)
pigments, v0.37.0 (active)
Remote-FTP, v0.9.1 (active)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.3.2 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.2.0 (inactive)
base16-tomorrow-light-theme, v1.2.0 (inactive)
one-dark-ui, v1.6.0 (inactive)
one-light-ui, v1.6.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.7.0 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.201.1 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.59.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v1.0.0 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2-hotfix1 (active)
snippets, v1.0.2 (active)
spell-check, v0.68.2 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.101.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.209.3 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by Moonbeam4520** [comment]
same issue : 
1.  Right click on the folder you want to create a file within
2. Click Add file , type a name and press enter..

**Atom Version**: 1.11.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.12.1
**Thrown From**: [Remote-FTP](https://github.com/mgrenier/remote-ftp) package, v0.9.1

### Stack Trace

Uncaught TypeError: Cannot read property 'remote' of undefined

```
At /Users/joelcostamagna/.atom/packages/Remote-FTP/lib/menus/commands.js:175

TypeError: Cannot read property 'remote' of undefined
    at space-pen-div.<anonymous> (/Users/joelcostamagna/.atom/packages/Remote-FTP/lib/menus/commands.js:175:38)
    at space-pen-div.jQuery.event.dispatch (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:4435:9)
    at space-pen-div.elemData.handle (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:4121:28)
    at Object.jQuery.event.trigger (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:4350:12)
    at space-pen-div.<anonymous> (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:4901:17)
    at Function.jQuery.extend.each (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:374:23)
    at AddDialog.jQuery.fn.jQuery.each (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:139:17)
    at AddDialog.jQuery.fn.extend.trigger [as originalTrigger] (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/jquery/dist/jquery.js:4900:15)
    at AddDialog.$.fn.trigger (/Users/joelcostamagna/.atom/packages/Remote-FTP/node_modules/space-pen/lib/space-pen.js:606:19)
    at AddDialog.module.exports.AddDialog.onConfirm (/Users/joelcostamagna/.atom/packages/Remote-FTP/lib/dialogs/add-dialog.js:22:8)
    at space-pen-div.atom.commands.add.core:confirm (/Users/joelcostamagna/.atom/packages/Remote-FTP/lib/dialogs/dialog.js:51:10)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:114:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -9:57.6.0 application:open (atom-pane.pane.active)
     -9:47.9.0 remote-ftp:connect (atom-pane.pane.active)
     -9:44.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -9:34.8.0 remote-ftp:add-file (span.name.icon.icon-file-directory)
  3x -9:31.6.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -9:29.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -8:58.1.0 remote-ftp:add-file (span.name.icon.icon-file-text)
     -8:46.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -8:31 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -7:09.6.0 settings-view:open (atom-text-editor.editor.is-focused)
     -6:40.5.0 atom-beautify:help-debug-editor (atom-text-editor.editor.is-focused)
     -5:36.2.0 settings-view:open (atom-text-editor.editor.is-focused)
     -0:06.7.0 remote-ftp:add-file (span.name.icon.icon-file-directory)
  5x -0:04.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:00.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

``` json
{
  "core": {
    "disabledPackages": [
      "linter-pep8",
      "symbols-view"
    ],
    "openEmptyEditorOnStart": false,
    "packagesWithKeymapsDisabled": [
      "atom-beautify"
    ],
    "telemetryConsent": "limited",
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ],
    "useCustomTitleBar": true
  },
  "Remote-FTP": {
    "hideLocalWhenDisplayed": true
  }
}
```

### Installed Packages

``` coffee
# User
atom-autocomplete-php, v0.19.4 (active)
atom-beautify, v0.29.13 (active)
atom-material-syntax, v0.4.6 (active)
atom-material-syntax-light, v0.4.5 (inactive)
atom-material-ui, v1.3.6 (active)
auto-update-packages, v1.0.1 (active)
autocomplete-clang, v0.10.0 (active)
autocomplete-java, v1.2.5 (inactive)
autocomplete-oracle, v0.4.0 (active)
autocomplete-python, v1.8.7 (active)
ctags-status, v1.3.6 (active)
docblockr, v0.8.7 (active)
file-icons, v1.7.23 (active)
flatwhite-syntax, v0.8.0 (inactive)
java-generator, v2.2.0 (inactive)
java-importer, v1.4.1 (active)
language-latex, v1.0.0 (active)
language-oracle, v1.6.0 (active)
latex, v0.38.1 (inactive)
latextools, v0.8.5 (active)
linter, v1.11.18 (active)
linter-clang, v3.4.4 (active)
linter-javac, v1.9.4 (active)
linter-php, v1.3.1 (active)
pdf-view, v0.50.0 (active)
php-getters-setters, v0.5.5 (active)
pigments, v0.37.0 (active)
python-tools, v0.6.8 (active)
Remote-FTP, v0.9.1 (active)
sync-settings, v0.7.2 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.2.0 (inactive)
base16-tomorrow-light-theme, v1.2.0 (inactive)
one-dark-ui, v1.6.0 (inactive)
one-light-ui, v1.6.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.7.0 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.201.1 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.59.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v1.0.0 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2-hotfix1 (active)
snippets, v1.0.2 (active)
spell-check, v0.68.2 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.101.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.209.3 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by jimmaaay** [comment]
Sorry about that. This should be fixed in 0.9.2, which I've just pushed out. If you have any issues still comment and I'll reopen the issue.

**Comment by pooreal** [comment]
Updated to 0.9.2 and the problem is fixed
